### DISCUSSION TITLE
why do I ahave localized entitiy names and how do I revert this to english?

### DISCUSSION THREAD

**Discussion by MabuClou** [greghesp/ha-bambulab#461] [Q&A]
I am new to HomeAssistant and just setup HA with the BambuLabs Integration. But now I noticed, that all the entities of my printer have german localized names instead of the original english names. Like it is called "_temperatur_der_duse" instead of "_nozzle_temperature".
This makes it very difficult to use ready prepared solutions like dashboards. For example this page states it is only usable with english language: https://www.wolfwithsword.com/bambulab-home-assistant-dashboard/
How can I revert my Bambulabs-Integration back to the standard english names?

**Comment by AdrianGarside** [comment]
You should only get German if you configure your home assistant to be in German.

**Comment by HRackl** [comment]
Did you find a solution? My HA is in German but I want the Bambu Integration in English

**Reply by MabuClou** [reply]
Well, that is exactly my problem. Until now I do not have a solution. I am still learning how to setup and use HomeAssistant. 

But I found out that there is a directory with translation files:  
_\ config \ custom_components \ bambu_lab \ translations_
inside there is a **de.json** and an **en.json**. This looks promising.
